duckduckgo/iOS
# [DuckPlayer] 5. First batch of pixels

Task/Issue URL: https://app.asana.com/0/1204099484721401/1207768980322967/f

**Description**:
Implements the first batch of pixels for DuckPlayer, incluiding: 

- m_duck-player_daily-unique-view
- m_duck-player_view-from_youtube_main-overla
- m_duck-player_view-from_youtube_automatic
- m_duck-player_view-from_serp
- m_duck-player_view-from_other
- m_duck-player_setting_always_settings
- m_duck-player_overlay_youtube_impressions
- m_duck-player_overlay_youtube_watch_here
- m_duck-player_setting_always_duck-player
- m_duck-player_setting_back-to-default
- m_watch-in-duckplayer_initial_u




**Steps to test this PR**:
1. CI should be 🥬
2. Smoke test the app confirm DDG pixels are fired

<!--
Before submitting a PR, please ensure you have tested the combinations you expect the reviewer to test, then delete configurations you *know* do not need explicit testing.
